src/main/java/sg/ncl/MainController.java

```diff
@@ -248,11 +282,16 @@ public String maintainance() {
         return "maintainance";
     }
 
-    @RequestMapping("/TestbedInformation")
+    @RequestMapping("/testbedInformation")
     public String TestbedInformation() {
         return "TestbedInformation";
     }
 
+    @RequestMapping("/testbedInformation1")
+    public String TestbedInformation1() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)